docs/server-side/v2/api-reference/analytics/track-events.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Track Events
+nav: 3.62
+---
+
+Track Events can be used to understand the participant's activity during the session, It would get you the reference points in time
+e.g. when the user has unmuted audio, shared screen, or stopped the video. Every event has a `mute` property to determine mute status at the time of the event.
+
+Following are the three types of events published for a track,
+
+**track.add.success**: Published when a track is added
+
+**track.update.success**: Published when a user has unmuted/muted the track.
+
+**track.remove.success**: Published when the track has ended
+
+Kind of a track (e.g. audio, video, video screenshare) can be determined by `type` and `source` parameter in the [track event object](/server-side/v2/api-reference/analytics/track-events#track-event-object).
+
+<GetRequest title="https://api.100ms.live/v2/analytics/events" />
+<Request id="req-0">
+
+```bash
+curl --location --request GET 'https://api.100ms.live/v2/analytics/events' \
+--header 'Authorization: Bearer <management_token>'
+```
+
+</Request>
+
+## Allowed Filters
+To be specified as query parameters
+
+| Name       | Type     | Description                                                                                                                                                                                                                                             | Required |
+| ---------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| type       | `string` | Type of events to be fetched (valid values `track.add.success`, `track.update.success`, `track.remove.success`) <br /><br /> Can be specified mulitple times to include more than one event (e.g. `track=track.add.success&` `track=track.update.success`) | yes      |
+| room_id    | `string` | Unique identifier of the room you wish to fetch events for                                                                                                                                                                                              | yes      |
```

Comment:
does this mean that i can't use the session id alone to query this API?

Reply:
yeah, `room_id` is mandatory.